backend/main/models/proposal.go

```diff
@@ -176,6 +188,31 @@ func (p *Proposal) UpdateProposal(db *s.Database) error {
 	return err
 }
 
+func (p *Proposal) UpdateDraftProposal(db *s.Database) error {
+	_, err := db.Conn.Exec(db.Context, `
+		UPDATE proposals
+		SET name = COALESCE($1, name), 
+		choices = COALESCE($2, choices),
+		strategy = COALESCE($3, strategy),
+		min_balance = COALESCE($4, min_balance),
+		max_weight = COALESCE($5, max_weight),
+		start_time = COALESCE($6, start_time),
+		end_time = COALESCE($7, end_time),
+		body = COALESCE($8, body),
+		block_height = COALESCE($9, block_height),
+		cid = COALESCE($10, cid),
+		composite_signatures = COALESCE($11, composite_signatures),
```

Comment:
i wonder if we should be updating composite_signatures/voucher fields.... in the long run it would be nice to collect and store a signature for each change that is made to the proposal to have a proper audit trail, but in lieu of that, we should just save the initial signature/voucher that was submitted to create the draft proposal in the first place, so its consistent.

obviously we should still validate the signature/voucher in the update API, but i dont think we should overwrite the original signatures in the DB